lib/ansible/_vendor/__init__.py

```diff
@@ -0,0 +1,19 @@
+# mask vendored content below from being accessed as an ansible subpackage, patch our vendored dir onto sys.path
+__path__ = []
+
+import pkgutil
+import os
+import sys
+
+vendor_path = os.path.dirname(__file__)
+
+vendored_module_names = set(m[1] for m in pkgutil.iter_modules([vendor_path], ''))
+already_loaded_vendored_modules = set(sys.modules.keys()).intersection(vendored_module_names)
+
+if vendored_module_names:
+    # patch us early to load vendored deps transparently
+    sys.path.insert(0, vendor_path)
+
+    print('hijacking sys.path to use ansible-vendored files, bwahaha')
```

Comment:
```suggestion
    print('hijacked sys.path to use ansible-vendored files, bwahaha')
```
(because this log entry is added after the fact)